satellite/console/usagerollups.go

```diff
@@ -35,7 +35,7 @@ type BucketUsage struct {
 
 	Storage     float64
 	Egress      float64
-	ObjectCount float64
+	ObjectCount uint64
```

Comment:
Not sure if this applies to this convention but we seem to prefer using `int` over unsigned integer https://github.com/storj/docs/blob/master/code/Style.md#integers

Reply:
seems, you are right